src/queries/types.ts

```diff
@@ -34,6 +34,7 @@ export const QUERY_TYPES = {
     USER_SETTINGS: 'userSettings',
     USER: 'consulUser'
   },
+  STATIC_CONTENT_LIST: 'contentList',
```

Comment:
to follow our pattern it should be `STATIC_CONTENT_LIST: 'staticContentList'`, right?

Reply:
Ow sorry. That's right!

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/958/commits/e5a5f952c1f59ab035f2cd42521c863ee3391068